contact/models.py

```diff
@@ -33,27 +33,47 @@ class Meta:
             models.Index(fields=["topic"]),
         ]
 
-    topic = models.CharField(max_length=50, verbose_name=translate("Sujet"), choices=TOPICS, default=TOPIC_CONTACT)
-    name = models.CharField(max_length=255, verbose_name=translate("Votre nom"), null=True, blank=True)
+    topic = models.CharField(max_length=50, verbose_name=translate_lazy("Sujet"), choices=TOPICS, default=TOPIC_CONTACT)
+    name = models.CharField(max_length=255, verbose_name=translate_lazy("Votre nom"), null=True, blank=True)
     email = models.EmailField(
         max_length=255,
-        verbose_name=translate("Adresse email"),
+        verbose_name=translate_lazy("Adresse email"),
         error_messages={"invalid": ""},
     )
     body = models.TextField(
         max_length=5000,
-        verbose_name=translate("Message"),
+        verbose_name=translate_lazy("Message"),
     )
     user = models.ForeignKey(
         settings.AUTH_USER_MODEL,
-        verbose_name=translate("Utilisateur"),
+        verbose_name=translate_lazy("Utilisateur"),
         null=True,
         blank=False,
         on_delete=models.SET_NULL,
     )
     erp = models.ForeignKey(Erp, null=True, blank=False, on_delete=models.SET_NULL)
-    sent_ok = models.BooleanField(verbose_name=translate("Envoi OK"), default=False)
+    sent_ok = models.BooleanField(verbose_name=translate_lazy("Envoi OK"), default=False)
 
-    # Datetimes
-    created_at = models.DateTimeField(auto_now_add=True, verbose_name=translate("Date de création"))
-    updated_at = models.DateTimeField(auto_now=True, verbose_name=translate("Dernière modification"))
+    created_at = models.DateTimeField(auto_now_add=True, verbose_name=translate_lazy("Date de création"))
+    updated_at = models.DateTimeField(auto_now=True, verbose_name=translate_lazy("Dernière modification"))
+
+
+class FAQ(models.Model):
+    SECTION_GENERAL = "general"
+    SECTION_CONTRIB = "contrib"
+    SECTION_DATA = "data"
+    SECTION_CHOICES = (
+        (SECTION_GENERAL, translate_lazy("Général")),
```

Comment:
On the figma, it's not in sync with the word document, we have the following wording: 
"Utilisation générale d’acceslibre", we may have to double check with Sophie & Perrine